### ISSUE TITLE
Fix roster <> profile interactions

### ISSUE THREAD

**Issue by marimuraki** [codeforamerica/somerville-teacher-tool#214]
**Issue with roster <> profile click back flow**

1. Add additional variables to roster view
2. Click on student to go to his/her profile view
3. If click back button --- which many teachers did (not "Back to Roster), takes you back to roster view but without the new added variables show while still listed in the selection box
4. If click "Back to Roster", at least it will take you to the default page without the confusion of showing the selected additional variables but not displaying it in table.

**Example**: "SPED & Disability" selected and was initially show in the roster but after going to the profile and hitting back, the columns are no longer there though the selection is still listed in text box

<img width="1076" alt="screenshot 2015-08-04 11 26 53" src="https://cloud.githubusercontent.com/assets/1334815/9064838/6b46a41e-3a9c-11e5-8bdf-ee125ca3ab17.png">

@alexsoble @AmirHadjihabib

**Comment by marimuraki** [comment]
@alexsoble @AmirHadjihabib: The state of the roster e.g., sort order, variable inclusion, etc is not maintained after returning to roster from profile.

GIF shows:

1. roster > variable selection > profile > back button > roster: variable selection is listed in text field but not shown in table
2. roster > variable selection > profile > "Back to Roster" > roster: default view without selected variable 
3. roster > sort > profile > roster: default view without sort

We need to fix this interaction as it adds unnecessary repetitions for teachers to "re-do" sorts, adds, etc and confuses the user. Comments / feedback:

- *"What happened to the table I just sorted?"*
- *"Why isn't SPED shown any more? I thought I added it."*
- *"Why is SPED shown in here but not in the table?*

![licecap_smvtt_roster_profile_interaction](https://cloud.githubusercontent.com/assets/1334815/9072336/9262b796-3ac8-11e5-99d5-e63d9aeb5b35.gif)
